### PR TITLE
[Tsql] Fix grammar for alter_service

### PR DESCRIPTION
**Pull request by jiangtao69039**
Add parentheses and keyword 'CONTRACT'.

Syntax from the documentation:
```
ALTER SERVICE service_name   
   [ ON QUEUE [ schema_name . ]queue_name ]   
   [ ( < opt_arg > [ , ...n ] ) ]  
[ ; ]  
  
<opt_arg> ::=  
   ADD CONTRACT contract_name | DROP CONTRACT contract_name
```

### CODE REVIEW

File: sql/tsql/TSqlParser.g4 [hunk 1]
```diff
@@ -1569,7 +1569,7 @@ alter_server_role_pdw
 
 // https://docs.microsoft.com/en-us/sql/t-sql/statements/alter-service-transact-sql
 alter_service
-    : ALTER SERVICE modified_service_name=id_ (ON QUEUE (schema_name=id_ DOT) queue_name=id_)? (COMMA? (ADD|DROP) modified_contract_name=id_)*
+    : ALTER SERVICE modified_service_name=id_ (ON QUEUE (schema_name=id_ DOT)? queue_name=id_)?  ('(' (COMMA? (ADD|DROP) CONTRACT modified_contract_name=id_)* ')')?
```
Comment (pguyot) [inline sql/tsql/TSqlParser.g4:1572]: I believe this rule accepts a leading comma:

```
ALTER SERVICE [//Adventure-Works.com/Expenses] ( , ADD CONTRACT [//Adventure-Works.com/Expenses/ExpenseSubmission] ) 
                                                ^^^
```

Could you please fix it?

Comment (jiangtao69039) [reply sql/tsql/TSqlParser.g4]: ok i fixed it.

**Comment by KvanTTT** [comment]
@pguyot is it ok to merge?

**Comment by pguyot** [comment]
Yes, it is. Thank you @jiangtao69039

**Pull request merged**